src/components/BankDetails/BankDetails.tsx

```diff
@@ -1,9 +1,10 @@
 import { useState } from "react";
 import { IFormButtons, OnSubmit } from "./types";
+import { Currency } from "types/components";
```

Comment:
@SovereignAndrey, opening this PR as preparatory for further redesign works. I unfortunately have made design UI changes (methods sidebar, ) before I discovered the need for restructuring. Let me know if I should exclude those to this PR.

Reply:
> Not commenting on the changes themselves as the code seems fine on first look, but the amount of not-necessarily-related changes in a single PR is very hard to review with any quality, it's difficult to keep track of everything.
Big PRs with many unrelated updates incentivize skimming and approving just to save time and effort, instead of actually reviewing the PR.

> @ap-justin would you at least consider breaking it up into smaller, more focused PRs? Would help make reviews easier and actually useful.

@misicnenad, changes generally falls into these sequential parts
1. donor tabs + common panels container - Towards `BG-1056`
2. I noticed that `stripe` and `paypal` has KYC form inside donate methods and to bring them out, some changes to `DonationState` should be made (also means standalone state inside stripe/paypal is now removed ) - Closes `BG-1094`
3. As paypal/stripe checkout is now under Submit step, make sure they are correctly rendered there (standalone) - so some spacing changes.